### ISSUE TITLE
PHP 8 null safe operator triggers ImportDetection.Imports.RequireImports.Symbol

### ISSUE THREAD

**Issue by underdpt** [squizlabs/PHP_CodeSniffer#3374]
**Describe the bug**

When using the null safe operator, the expression that follows the operator triggers ImportDetection.Imports.RequireImports.Symbol

**Code sample**
```php
$nullObject = null;
$variable = $nullObject?->property;
```

This throws
`
Found unimported symbol 'property'.
(ImportDetection.Imports.RequireImports.Symbol)
`

**Custom ruleset**
```xml
<?xml version="1.0"?>
<ruleset name="mainsip">
    <description>mainsip ruleset</description>

    <exclude-pattern>/deploy.php</exclude-pattern>
    <exclude-pattern>/ray.php</exclude-pattern>

    <rule ref="ImportDetection"/>
    <rule ref="ImportDetection.Imports.RequireImports">
        <properties>
            <property name="ignoreUnimportedSymbols" value="/^(__|abort_if|activity|app|asset|base_path|bcrypt|collect|csrf_token|config|dump|event|notify|redirect|report|request|response|route|Route|storage_path|test|throw_if|trans|view)$/"/>
        </properties>
    </rule>

</ruleset>

```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```
$ vendor/bin/phpcs test.php 

FILE: /home/vagrant/enertrade/intranet/test.php
----------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
----------------------------------------------------------------------
 4 | WARNING | Found unimported symbol 'property'.
----------------------------------------------------------------------

Time: 39ms; Memory: 10MB
```

**Expected behavior**

It should not trigger ImportDetection.Imports.RequireImports.Symbol, the expression next right to the null safe operator is not a symbol.

**Versions (please complete the following information):**
 - OS: Windows 10, Ubuntu 20.04
 - PHP: 8.0.5
 - PHPCS: 3.6.0
 - Standard:

**Comment by jrfnl** [comment]
@underdpt The `ImportDetection` standard is an external standard and not part of PHPCS. Please report your issue to thaat repo: https://github.com/sirbrillig/phpcs-import-detection

**Comment by underdpt** [comment]
Sorry!
